src/ui/tableOfContents.js

```diff
@@ -15,42 +15,67 @@ class TableOfContents extends Component {
-                    {this.props.tags.map((tag, i) =>
-                        (<li key={i}
-                            onClick={() =>
-                                window.location.hash = `#/${hashConst.tag}/${tag.ID}`
-                            }>{tag.tagName}</li>),
-                    )}
-                </ul>
-
-                <div className={"well well-sm"}>
-                    <h4>Rules</h4>
-                </div>
-                <div className={"ist-inline"}>
-                    <table>
-                        <tbody>
-                            {this.props.rules.map((rule, i) =>
-                                <tr key={i}>
-                                    <td className={"list-group-item"}>{rule.title}</td>
-                                </tr>,
-                            )}
-                        </tbody>
-                    </table>
-                    <div style={{padding: "10px 0 10px 0", clear: "both"}}>
-                        <Button style={{padding: "0 5px"}}
-                            onClick={() => {
-                                window.location.hash = `#/${hashConst.rules}`;
-                                this.props.onAddNewRule();
-                            }}>
-                            <MdPlaylistAdd className={"react-icons"} size={35}/>
-                            Add a New Rule
-                        </Button>
+                {this.props.rules.length > 0 ? (
+                    <>
+                        <div className={"well well-sm"}>
+                            <h4>Rules</h4>
+                        </div>
+                        <div className={"ist-inline"}>
+                            <table>
+                                <tbody>
+                                    {this.props.rules.map((rule, i) => (
+                                        <tr key={i}>
+                                            <td className={"list-group-item"}>{rule.title}</td>
+                                        </tr>
+                                    ))}
+                                </tbody>
+                            </table>
+                        </div>
+                    </>
+                ) : (
+                    <div>
+                        <p>"Welcome to the Rules Dashboard! This tool helps you manage design rules for your projects."</p>
+                        <p>Explore how easy it is to create, view, and manage design rules with the intuitive interface.</p>
+                        <br></br>
+                        <div style={{marginTop: "20px"}}>
+                                <h4>How Active Documentation Works ?</h4>
+                                <p>
+                                    ActiveDocumentation seamlessly integrates into your IDE and codebase, automatically monitoring and enforcing design rules.
+                                </p>
+                                <h5>Components</h5>
```

Comment:
This part might be misleading since the page is already open in the web app. 
I suggest chaning it to mention that a plugin needs to be installed in your IDE, with a link explaining how to do that.